docs/tasks/announcements.rst

```diff
@@ -1,46 +1,87 @@
+=============
 Announcements
 =============
 
-You mean there's more than place for announcements?
----------------------------------------------------
+As of 2018, there is only one place to write announcements: `CampusGroups <campusgroups.rit.edu>`__.
+Announcements are generally sent to RITlug club members at the beginning of each week.
+These weekly emails inform club members about the topic of the upcoming general meeting.
+
+
+How to Create an Announcement
+=============================
+
+In order to post a club-wide announcement, you need to navigate to the RITlug `CampusGroups <https://campusgroups.rit.edu>`__ page.
+On the main club page, go to the *Emails* drop-down, and select the orange *Compose Email* button.
+To send an email from here, one must:
+
+#. Add and review list of recipients
+#. Choose sender email address
+#. Select email template
+#. Compose email
+#. Send email
+
+Selecting Recipients
+--------------------
+
+Here, you will be asked which group(s) you would like to send the email to.
+
+.. figure:: /_static/img/campusgroups_email_recipients.png
+   :alt: CampusGroups Select Recipients Screen
+
+   CampusGroups Select Recipients Screen
+
+Choose a "From" Email Address
```

Comment:
Please add a note that this is technically a "Reply-To" header despite CampusGroups calling it "From"

Reply:
I think the key thing to emphasize is what appears in the CampusGroups UI, so there is no confusion over what the correct setting to change is. If adding a note, it should be clearly indicated as a technicality and de-emphasized in the doc (perhaps by using bold on what does appear in the UI).

Reply:
That's just what CampusGroups calls it so that's what I thought made the most sense. I'm not sure if it's necessary to add in the technicality, but I will definitely look into that, thanks!

Reply:
It is a technical thing, but it's really important for spam filter debugging

Reply:
@ct-martin I've added a note for the debugging